editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -5901,6 +5901,15 @@ void CanvasItemEditorViewport::_create_preview(const Vector<String> &files) cons
 					Node *instance = scene->instance();
 					if (instance) {
 						preview_node->add_child(instance);
+
+						Node2D *n = Object::cast_to<Node2D>(instance);
+						if (n) {
+							n->set_position(Vector2());
```

Comment:
Does not _edit_set_position() work for both ?

Reply:
It doesn't work for Control when anchor-only mode is enabled.